.circleci/config.yml

```diff
@@ -98,27 +46,6 @@ jobs:
       - install_dependencies
       - run: composer phpcs
 
-  deploy_svn_branch:
```

Comment:
Not related, just cleanup. We're not deploying on merging into `master` anymore, just on tagging